### PR TITLE
Updated target framework, ini parser-dependency + minor refactoring.

### PR DESCRIPTION
**Pull request by staalezh**

### CODE REVIEW

File: Lucidtech/Las/Core/Credentials.cs [hunk 1]
```diff
@@ -121,37 +121,41 @@ private object JsonDecode(IRestResponse response)
...
             string apiEndpoint
-        ) {
+        ) : this()
+        {
             ClientId = clientId;
             ClientSecret = clientSecret;
             AuthEndpoint = authEndpoint;
             ApiEndpoint = apiEndpoint;
-            CommonConstructor();
+            //CommonConstructor();
```
Comment (magnurud) [inline Lucidtech/Las/Core/Credentials.cs:130]: remove this comment

Comment (magnurud) [reply Lucidtech/Las/Core/Credentials.cs:130]: Can we remove the method CommonConstructor now?

**Review by magnurud — CHANGES_REQUESTED** [review]

File: Lucidtech/Las/Core/Credentials.cs (hunk 1 above)
Comment (staalezh) [reply Lucidtech/Las/Core/Credentials.cs:130]: Yup - thanks for noticing!

**Review by magnurud — APPROVED** [review]

**Review by magnurud — COMMENTED** [review]
Remember to update the changelog!

**Pull request merged**